src/store/posts/index.ts

```diff
@@ -0,0 +1,79 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+// import axios from 'axios';
+
+import hireItems from '@/fixtures/hireItems';
+import matchItems from '@/fixtures/matchItems';
+
+export interface PostWrapper {
+  isMatch: boolean;
+}
+
+export interface Post {
+  item: PostItem;
+}
+
+export interface PostItem {
+  matchId?: number;
+  postId?: number;
+  city: string;
+  region: string;
+  groundName: string;
+  date: string;
+  startTime: string;
+  cost?: number;
+  position?: string;
+  ageGroup: string;
+  teamLogo: string;
+  teamName: string;
+  teamMannerTemperature: string;
+}
+
+export const fetchAllPost = createAsyncThunk('posts/fetchAllPost', async () => {
+  /**
+   * Todo: API 완성시 추가
+   * const { data } = await axios.get(URL);
+   * return data;
+   */
+
+  return hireItems;
+});
+
+export const fetchAllMatch = createAsyncThunk('matches/fetchAllMatches', async () => {
+  /**
+   * Todo: API 완성시 추가
+   * const { data } = await axios.get(URL);
+   * return data;
+   */
+
+  return matchItems;
+});
+
+export interface PostsState {
+  data: PostItem[];
+}
+
+export const posts = createSlice({
+  name: 'posts',
+  initialState: {
+    data: [],
+  } as PostsState,
```

Comment:
```suggestion
  initialState: {
    data: {
      matchItems: [],
      hireItems: [ ]
    },
  } as PostsState,
```
상태에 용병과 매치에 관련한 데이터를 따로 관리하기 위하여
이런식으로 관리하여 extraReducers도 수정할 예정입니다.

Reply:
matches: [ ] 매치리스트
match: [ ]  현재 매치 페이지

Reply:
리덕스는 그때 토론한것대로 이 부분에서는 전역상태관리를 할 필요성이 없는것 같아서 추후 변경하겠습니다.